### ISSUE TITLE
Show average running time of each step while running

### ISSUE THREAD

**Issue by twe4ked** [buildkite/feedback#409]
It would be really nice to be able to see the average running time for a certain step while the step is still running.

E.g. Next to “4m 22s” in this screenshot. Or as a `title` attribute or similar.

<img width="1187" alt="screen shot 2018-06-28 at 2 59 11 pm" src="https://user-images.githubusercontent.com/71970/42014236-8c348872-7ae4-11e8-9c2c-0373eb22192b.png">

Thanks!

**Comment by pauldraper** [comment]
Agreed. Jenkins provides an ETA on builds based on previous run times.
